src/main/java/org/skriptlang/skript/bukkit/spawner/elements/expressions/spawnerentry/equipment/ExprEquipmentWithLootTable.java

```diff
@@ -0,0 +1,58 @@
+package org.skriptlang.skript.bukkit.spawner.elements.expressions.spawnerentry.equipment;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.loot.LootTable;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.spawner.SpawnerModule;
+import org.skriptlang.skript.bukkit.spawner.util.SpawnerEntryEquipment;
+
+@Name("Spawner Entry - Equipment with Loot Table")
+@Description("Returns the equipment loot table of a spawner entry equipment.")
+@Examples("set {_loot table} to loot table of {_equipment}")
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.21+")
+public class ExprEquipmentWithLootTable extends SimplePropertyExpression<SpawnerEntryEquipment, LootTable> {
+
+	static {
+		register(SpawnerModule.SYNTAX_REGISTRY, ExprEquipmentWithLootTable.class, LootTable.class,
+			"loot[ ]table", "spawnerentryequipments");
```

Comment:
is this not going to conflict with other loottable syntaxes? Like `loottable of block`???